pa4/checker-pa4.ml

```diff
@@ -258,6 +264,22 @@ let add_attribute_types o input_class = begin
 end;;
 
 
+
+let add_method_types meth_env input_class = begin
```

Comment:
Is this supposed to be retrieving the return types of the methods or the types of their parameters? Right now it looks like it returns the type of it's parameters rather than the method's return type, so the name of this method should be clarified.

Reply:
it doesn't retrieve or return anything. it just adds method types to method environment.

Reply:
Looking down at the rest of the code, I realize that now. Thanks for the clarification.